### PR TITLE
added sql code for sitter application table

### PR DESCRIPTION
**Pull request by Lior-Becker**
Note it does need the user and bookings table to exist as there are foreign key dependencies

### CODE REVIEW

**Comment by timoBBD** [comment]
The code and naming convention is good👍 But you can rename the file to be V1. The current V1 is just an example that we will remove before applying the migrations

**Comment by Lior-Becker** [comment]
> The code and naming convention is good👍 But you can rename the file to be V1. The current V1 is just an example that we will remove before applying the migrations

it will need to become v3 or later at some point because its reliant on 2 other tables, but ill call it v1 in the mean time

**Review by jennadunfordbbd — APPROVED** [review]
Looks good to me! 👍

File: flyway/migrations/V8__createSitterApplicationTable.sql
Comment (RotendaBBD) [inline flyway/migrations/V8__createSitterApplicationTable.sql:1]: Everything Looks fine

**Pull request merged**